cdk/src/functions/service-control-policies-sandbox.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  AttachPolicyCommand,
+  CreatePolicyCommand,
+  DeletePolicyCommand,
+  DetachPolicyCommand,
+  ListRootsCommand,
+  OrganizationsClient,
+  PolicyType,
+  UpdatePolicyCommand,
+  ListPoliciesCommand,
+  ListOrganizationalUnitsForParentCommand,
+} from '@aws-sdk/client-organizations';
+import { CdkCustomResourceEvent, CdkCustomResourceResponse, Context } from 'aws-lambda';
+import { throttlingBackOff } from './utils/throttle';
+
+async function getSandboxId(organizationClient: OrganizationsClient): Promise<string | undefined> {
```

Comment:
this will not work for all superwerker installations. older ones have a different naming scheme where the sandbox ou was previously called custom. you must read out the /superwerker/controltower/sandbox_ou_name parameter to get the correct name of the OU